coverage.json

```diff
@@ -0,0 +1,18 @@
+
+{
+    "debug":{
+        "options" : {
+            "coverage" : {
+                "modules" : {
+                    "mbed-drivers" : true
+                }
+            }
+        }
+    },
+    "mbed-os": {
```

Comment:
It improves readability, same applies to our "code" files, we should care :facepunch: